[TASK] Overhaul backend login page (#306)
TYPO3-Documentation/TYPO3CMS-Tutorial-Editors: Documentation/Login/Index.rst

Logging in to TYPO3 backend
===========================

If you are a TYPO3 editor, the backend is where you will most likely spend most
of your time. So let's start by logging in.

> Navigate to  :samp:`https://www.yourdomain.org/typo3` in your browser. The

Reviewer: **Use `example.org` instead of `yourdomain.org`**

TYPO3 docs maintain a fixed set of hypothetical/placeholder domains for exactly this situation — a URL where the domain itself doesn't matter, only its role in the example. Per [the guide](https://docs.typo3.org/permalink/h2document:preventing-links), the set is, in preference order:

1. `https://example.org` (preferred)
2. `https://example.com` (if a second domain is needed)
3. `https://example.net` (if a third is needed)

`.org` is preferred over `.com` specifically because it matches TYPO3 docs being hosted on `typo3.org` and the TYPO3 Association's ownership of TYPO3 GmbH.

The important part isn't just style — `example.org` is one of the domains reserved by IANA for documentation use, so it's guaranteed to never resolve to a real, possibly-unrelated website. `yourdomain.org` carries no such guarantee.

Suggested fix:
```suggestion
Navigate to :samp:`https://example.org/typo3` in your browser. The
```

(Also note the `:samp:` role is already used correctly here — that's the same mechanism the guide recommends to stop the renderer from auto-linking the URL, so no change needed there.)

#.  Enter your email address

    Use the same email address that you entered during registration.

    ..  figure:: /Images/ManualScreenshots/Login/ForgotPasswordEmail.png

>         :alt: Click on "Forgot your password?"

Reviewer: **Alt text describes the wrong image**

This `:alt:` text is unchanged from before this PR, but the screenshot itself was replaced (`ForgotPasswordEmail.png` went from 18KB to 34KB) and the caption below was updated to match ("Enter your email address"). The alt text still says `Click on "Forgot your password?"`, which described the *previous* screenshot (the login page with that link highlighted) — not what's actually embedded now.

For accessibility, alt text should describe what the image actually shows (someone using a screen reader can't see the caption's visual context), per [Guidelines for creating images](https://docs.typo3.org/permalink/h2document:guidelines-for-images): *"For accessibility reasons **always** provide an alt text."* Right now it just doesn't match the image anymore.

Suggested fix — something describing the actual current screenshot, e.g.:
```suggestion
        :alt: The password reset form with the email address field
```
(adjust wording to match what the screenshot actually shows)

        :zoom: gallery

        Enter your email address

#.  Open the email and follow the link

    If you haven't received the email, check your spam folder and
    double-check that the email address you entered is correct.

#.  Enter a new password

    Click on the password recovery link in the email and then enter a new
    password. Make sure you choose a password that fits your administrator's
    rules.

Things your **administrator** can check if this is not working correctly:

*    the :ref:`password reset feature<t3coreapi:access-password-reset>` is
     enabled
*    your backend account has a valid email address associated with it

> *    the TYPO3 installation can send emails.

Reviewer: **List marker spacing is off by one space**

This list uses `*` + 4 spaces (list text starts at column 5), but the [reST coding guidelines](https://docs.typo3.org/permalink/h2document:cgl-indenting) specify:

> list markers are followed by enough spaces to line up item text at a 4-space column: 3 spaces after a single-character marker (`*`, `-`)

So it should be `*` + 3 spaces (text at column 4) — which is exactly what the other bullet list further down in this same file (lines 75–77, `*   JavaScript is enabled...`) already does correctly. The reason for the 4-space column: continuation lines of a list item need to line up with the item text, and 4 spaces is this project's standard indentation unit everywhere else in the file too.

Handy trick for editors: if you type `*`, then a space, then hit Tab, most editors (and reST-aware ones especially) will expand that to the next 4-space stop for you — landing you on `*` + 3 spaces automatically, no manual counting needed.

```suggestion
*   the :ref:`password reset feature<t3coreapi:access-password-reset>` is
    enabled
*   your backend account has a valid email address associated with it
*   the TYPO3 installation can send emails.
```
